### DISCUSSION TITLE
Show tags in completion

### DISCUSSION THREAD

**Discussion by RaZ0rr-Two** [hrsh7th/nvim-cmp#574] [Q&A] [answered]
Is there an easy way to integrate `:h i_CTRL-X_CTRL-]` ( `compl-tag` ) choices in cmp completion window/list ?

Or is the easy way to use [cmp-nvim-tags](https://github.com/quangnguyen30192/cmp-nvim-tags) ?

**Comment by hrsh7th** [comment, marked as answer]
Yes. You should use cmp-nvim-tags.
